src/components/AccountSettings.tsx

```diff
@@ -331,7 +330,26 @@ export const AccountSettings = () => {
                       </span>
                     )}
                   </div>
-                  <ORCiDButton sx={credentialButtonSX} />
+                  {orcid ? (
+                    <Button
+                      variant="outlined"
+                      sx={credentialButtonSX}
+                      onClick={() => setShowUnbindORCiDDialog(true)}
+                    >
+                      Unlink your ORCID profile
+                    </Button>
+                  ) : (
+                    <ORCiDButton
+                      sx={credentialButtonSX}
+                      redirectAfter={`${SynapseClient.getRootURL()}authenticated/myaccount`}
```

Comment:
Used to redirect to the default path, which was in the Profile Validation workflow.  Need to set this to go back to this account page.